frontend/.gitignore

```diff
@@ -21,3 +22,8 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+
+notes.local.md
+/build
+/public/build
```

Comment:
Do we have `public/build` at all?

Reply:
regarding ```public/build``` this can be removed. This is inserted during testing with tailwindcss